### ISSUE TITLE
Underscore for last expression is not working

### ISSUE THREAD

**Issue by caternuson** [adafruit/circuitpython#480]
Enabled in MicroPython via MICROPY_CAN_OVERRIDE_BUILTINS flag.

But in CP:
```python
Adafruit CircuitPython 2.1.0 on 2017-10-17; Adafruit Trinket M0 with samd21e18
>>> import time
>>> time.monotonic()
84728.1
>>> _
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name '_' is not defined
>>>

**Comment by dhalbert** [comment]
Uses 168 bytes. Implemented via the override-builtins dict, hence the dependence on that flag.

**Comment by dhalbert** [comment]
This is turned on in nrf and esp8266 ports. Let's turn it on in atmel-samd for consistency.

**Comment by dhalbert** [comment]
Fixed by #518.
